pytorch_toolkit/object_detection/tests/common/test_case.py

```diff
@@ -12,14 +12,17 @@
 # See the License for the specific language governing permissions
 # and limitations under the License.
 
+import logging
 import json
 import os
+from subprocess import run
 import unittest
 
 import torch
 import yaml
 
 from common.utils import collect_ap, run_through_shell
+from ote.misc import get_file_size_and_sha256, download_snapshot_if_not_yet
```

Comment:
`get_file_size_and_sha256` is unused import.

Reply:
removed